node/providers/anthropic.ts

```diff
@@ -581,7 +612,9 @@ export function placeCacheBreakpoints(messages: MessageParam[]): number {
           }
           break;
         case "document":
-          lengthAcc += block.source.data.length;
+          if ("data" in block.source) {
+            lengthAcc += block.source.data.length;
+          }
```

Comment:
the newest version of the sdk complained that block.source.data does not exist in `ContentBlockSource`